src/chrome/content/rules/Url2Png.xml

```diff
@@ -3,6 +3,9 @@
 	<target host="www.url2png.com" />
 	<target host="api.url2png.com" />
 
+	<!-- url2png.com cert has expired but redirects to www. anyway -->
+	<rule from="^http://url2png\.com" to="https://www.url2png.com" />
```

Comment:
Change to `<rule from="^http://url2png\.com/" to="https://www.url2png.com/" />` to fix the build error.